src/gamekit.cs

```diff
@@ -2215,49 +2215,63 @@ interface GKGameSession
 		[Export ("badgedPlayers")]
 		GKCloudPlayer[] BadgedPlayers { get; }
 
+		[Async]
 		[Static]
 		[Export ("createSessionInContainer:withTitle:maxConnectedPlayers:completionHandler:")]
 		void CreateSession (string containerName, string title, nint maxPlayers, Action<GKGameSession, NSError> completionHandler);
 
+		[Async]
 		[Static]
 		[Export ("loadSessionsInContainer:completionHandler:")]
 		void LoadSessions (string containerName, Action<GKGameSession[], NSError> completionHandler);
 
+		[Async]
 		[Static]
 		[Export ("loadSessionWithIdentifier:completionHandler:")]
 		void LoadSession (string identifier, Action<GKGameSession, NSError> completionHandler);
 
+		[Async]
 		[Static]
 		[Export ("removeSessionWithIdentifier:completionHandler:")]
 		void RemoveSession (string identifier, Action<NSError> completionHandler);
 
+		[Async]
 		[Export ("getShareURLWithCompletionHandler:")]
 		void GetShareUrl (Action<NSUrl, NSError> completionHandler);
 
+		[Async]
 		[Export ("loadDataWithCompletionHandler:")]
 		void LoadData (Action<NSData, NSError> completionHandler);
 
+		[Async]
 		[Export ("saveData:completionHandler:")]
 		void SaveData (NSData data, Action<NSData, NSError> completionHandler);
 
+		[Async]
 		[Export ("setConnectionState:completionHandler:")]
 		void SetConnectionState (GKConnectionState state, Action<NSError> completionHandler);
 
 		[Export ("playersWithConnectionState:")]
 		GKCloudPlayer[] GetPlayers (GKConnectionState state);
 
+		[Async]
 		[Export ("sendData:withTransportType:completionHandler:")]
 		void SendData (NSData data, GKTransportType transport, Action<NSError> completionHandler);
 
+		[Async]
 		[Export ("sendMessageWithLocalizedFormatKey:arguments:data:toPlayers:badgePlayers:completionHandler:")]
 		void SendMessage (string key, string[] arguments, NSData data, GKCloudPlayer[] players, bool badgePlayers, Action<NSError> completionHandler);
 
+		[Async]
 		[Export ("clearBadgeForPlayers:completionHandler:")]
 		void ClearBadge (GKCloudPlayer[] players, Action<NSError> completionHandler);
 
-		[Static]
-		[Export ("acceptShareURL:handler:")]
-		void AcceptShareUrl (NSUrl url, Action<NSError> handler);
+		//FIXME: Comment from header files:
+		//Remove after we get <rdar://problem/24623538> Game Center needs a way of defining an XPC interface for daemons
```

Comment:
That's a bit confusing. It's not commented in the headers, which sounds like the API will is temporary and will be removed later. In our case removing might sound like _uncomment_ the API

IIRC it did not work (at least respond) when executed - so it might already have been removed and the header is out of date.